recipes/k8/meta.yaml

```diff
@@ -1,22 +1,48 @@
-{% set version = "0.2.5" %}
+{% set version = "1.0" %}
 package:
   name: k8
   version: {{ version }}
 
 source:
-  url: https://github.com/attractivechaos/k8/releases/download/{{ version }}/k8-{{ version }}.tar.bz2
-  sha256: a937ac44532e042cd89ac743647b592c21cfcf31679e39e5f362e81034d93d18
+  url: https://github.com/attractivechaos/k8/releases/download/v{{ version }}/k8-{{ version }}.tar.bz2
+  sha256: 61504dad2d63404bf523d3f8d0a8bfd72ec78aa7bd79bdf9291a4f629cfb9c02
 
 build:
-  number: 4
-
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('k8', max_pin="x") }}
+  string: h3f4af3b_4
 requirements:
   build:
     - {{ compiler('cxx') }}
+    - _libgcc_mutex >=0.1
+    - _openmp_mutex >=4.5
+    - binutils_impl_linux-aarch64 2.40 h870a726_0
```

Comment:
This looks wrong to me. 
Why do you use `**-aarch64` dependencies here ? How is the x86_64 supposed to use them ?

Reply:
I added this tag. Doesn't it mean it 's only available in linux-aarch64's repository of conda?
<img width="351" alt="image" src="https://github.com/bioconda/bioconda-recipes/assets/144549629/97ed218f-4785-4501-945f-1476ba10c9c0">

Reply:
@martin-g I added this tag. Doesn't it mean it 's only available in linux-aarch64's repository of conda?
<img width="351" alt="image" src="https://github.com/bioconda/bioconda-recipes/assets/144549629/97ed218f-4785-4501-945f-1476ba10c9c0">

Reply:
This is `extra`, i.e. in addition to the default platforms.
You need to use selectors, like `... # [aarch64 or osx]` to specify that something is specific for some architecture